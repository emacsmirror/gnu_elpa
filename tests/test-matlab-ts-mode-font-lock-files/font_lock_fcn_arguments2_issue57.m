% -*- matlab-ts -*-

function font_lock_fcn_arguments2_issue57(a,b)
    arguments
        a (1,:) double {mustBeReal}
        b (1,:) double {mustBeReal, myUtils.mustBeSameSize(a, b)}
    end

    disp(a)
    disp(b)
end
