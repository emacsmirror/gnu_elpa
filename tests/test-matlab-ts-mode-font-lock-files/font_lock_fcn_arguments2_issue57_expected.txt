c ccc ccccccccc ccc

kkkkkkkk ffffffffffffffffffffffffffffffffbvDvb
    kkkkkkkkk
        P bnDDb tttttt bFFFFFFFFFFb
        P bnDDb tttttt bFFFFFFFFFFD FFFFFFFDFFFFFFFFFFFFFFwww wwb
    kkk

    BBBBbdb
    BBBBbdb
kkk
